app/views/dams_collections/show.html.erb

```diff
@@ -33,32 +32,34 @@
 
 <%# Main container %>
 <div class="row-fluid dams-collections">
-	<%# Corpus %>
+  <%# Corpus %>
   <div class="span8">
-	  <p class="crumb hidden-phone"><b>Collections</b> &raquo;</p>
+    <p class="crumb hidden-phone"><b>Collections</b> &raquo;</p>
     <div class="hidden-phone"><%= render :partial => 'shared/fields/title' %></div>
     <h3>About this collection
-    <%= render :partial => 'dams_objects/social_links' %></h3>
+      <%= render :partial => 'dams_objects/social_links' %></h3>
     <dl><%= render :partial => 'shared/fields/show_raw', :locals => {:greetingsFromCollectionsView => 'Wish you were here!'} %></dl>
   </div>
-	<%# Sidebar %>
+  <%# Sidebar %>
   <div class="span4">
-	  <%# Collections controls for desktop layout %>
+    <%# Collections controls for desktop layout %>
     <div class="hidden-phone">
-	    <%# Collection image %>
-	    <div id="collections-image" class="hidden-phone" style="background-image:url(http://libraries.ucsd.edu/assets/dams/home-page/img/<%=@document['id']%>.jpg),url(http://libraries.ucsd.edu/assets/dams/home-page/img/default.jpg)"></div>
-	    <%# Collection controls %>
-	    <% if @object_count > 0 %>
-	      <%= link_to "<i class='icon-eye-open icon-white'></i> View Collection Items".html_safe, facet_uri(@collection_title,'collection_sim'), :class => 'btn btn-large btn-inverse btn-block' %>
-	      <div class="well collections-search-box"><%= render :partial => 'search' %></div>
-	    <% else %>
-	      <p class="alert alert-no-dismiss"><%=no_collection_items_notice%></p>
-	    <% end %>
-	  </div>
+      <%# Collection image %>
+      <div id="collections-image" class="hidden-phone">
```

Comment:
This seems like the main testable change here -- can you add a feature test for any of these changes?